src/DOOM/p_spec.c

```diff
@@ -1008,6 +1008,9 @@ void P_PlayerInSpecialSector(player_t* player)
         case 9:
             // SECRET SECTOR
             player->secretcount++;
+            menuactive = false;
+            M_StartMessage("A secret is revealed!\n", doom_secretFound, false);
+            menuactive = false;
```

Comment:
Why the 2 `menuactive`?

Reply:
The function changes menu active iirc. I want to ensure it's not marked as active.